src/data/general-cs-quiz.ts

```diff
@@ -1,4 +1,59 @@
 const generalCSQuiz = [
+  {
+    Question:
+      "Which of the following is corect about a Tree?",
+    Answer: "Tree is minimally connected.",
+    Distractor1: "Tree can be cyclic.",
+    Distractor2: "Tree must be undirected graph",
+    Distractor3: "Tree must have atleast two nodes",
+    Explanation: 
+      "If we remove any edge from a tree, it creates a two disconnected graph",
+    Link: "https://www.youtube.com/watch?v=09_LlHjoEiY&list=PLWKjhJtqVAbn5emQ3RRG8gEBqkhf_5vxD&t=306s",
+  },
+  {
+    Question:
+      "Which of the following is a constraint for Dijkstra's Single Source Shortest Path Algorithm",
+    Answer: "The graph must only contain non-negative edge weights.",
+    Distractor1: "The graph must be acylic",
+    Distractor2: "The graph must be a balanced binary tree",
+    Distractor3: "The graph must have unique edge weights",
+    Explanation: 
+      "Dijkstra's SSSP algorithm may not work for a graph with negative edge weights. This constraint is to ensure that once a vertex is visited, the optimal distance from source cannot be updated further.",
+    Link: "https://www.youtube.com/watch?v=09_LlHjoEiY&list=PLWKjhJtqVAbn5emQ3RRG8gEBqkhf_5vxD&t=4832s",
+  },
+  {
+    Question:
+      "Which of the following is the most correct statement.",
+    Answer: "Topological sort is possible only for a Directed Acyclic Graph",
+    Distractor1: "Toplogical sort is just an another name for BFS",
+    Distractor2: "All undirected graph have topological ordering",
+    Distractor3: "A Cyclic graph also have topological ordering.",
+    Explanation: 
+      "The only graphs that has valid topological ordering is a Directed Acyclic Graph.",
+    Link: "https://www.youtube.com/watch?v=09_LlHjoEiY&list=PLWKjhJtqVAbn5emQ3RRG8gEBqkhf_5vxD&t=3632s",
+  },
+  {
+    Question:
+      "What is ASCII",
+    Answer: "It is a character encoding standard to respresent text in computer.",
+    Distractor1: "Algorithm to convert decimal to number in base 65.",
+    Distractor2: "A protocol in Layer 3 of TCP/IP Suite",
+    Distractor3: "A unit of throughput for a processor.",
+    Explanation: 
+      "American Standard Code for Information Interchange (ASCII) is a character encoding standard to respresent text in computer.",
+    Link: "https://www.youtube.com/watch?v=LtoBGQPuu1c&list=PLWKjhJtqVAbn5emQ3RRG8gEBqkhf_5vxD&t=406s",
+  },
+  {
+    Question:
+      "Which of the following is True?",
```

Comment:
I don't think true needs to be capitalized here

```suggestion
      "Which of the following is true for Big O?",
```